### PR TITLE
ACS - JOSS revisions for adding units and updated quickstart

### PR DESCRIPTION
**Pull request by abigailsnyder**
We do need to update the package data - I have a built version but only @kdorheim can upload new data to the zenodo and she isn't available until Wed Aug 23 .

### CODE REVIEW

File: notebooks/stitches-quickstart.ipynb
Comment (abigailsnyder) [inline notebooks/stitches-quickstart.ipynb:1]: The `install_package_data` is currently commented out so I could work with the updated package data I have locally - when @kdorheim is able to help get updated package data on zenodo, will need to un-comment and re-run quickstart for QC

**Comment by crvernon** [comment]
@kdorheim will you go ahead and upload the zenodo record and paste the link to the new DOI here?  I'll then add it into the code for the PR.  I run this no prob after that and then I'll approve.  Thanks!

**Comment by crvernon** [comment]
Test for this now failing due to data diff which is what should be expected in `feature/tests`. Once the new data is up, this should work.

**Comment by crvernon** [comment]
Closing to manage in `dev` as a consolidated effort

**Pull request merged**